im.riot.Riot.yaml

```diff
@@ -158,45 +153,40 @@ modules:
         commit: 245d13875faaecb7c3fec2df112bf3a7a6e8fec9
         dest: main
         x-checker-data:
-          type: git
-          tag-pattern: ^v([\\d.]+)$
-          version-scheme: semantic
+          type: json
```

Comment:
I'm just curious about why you choose the `json` `x-checker-data`?

Reply:
Because json checker can get stable timestamps, unlike git checker, see flathub/flatpak-external-data-checker#154